SECURITY.md

```diff
@@ -0,0 +1,66 @@
+# Security Policy
+
+ - [**Using AutoGPT Securely**](#using-AutoGPT-securely)
+   - [Restrict Workspace](#restrict-workspace)
+   - [Untrusted inputs](#untrusted-inputs)
+   - [Data privacy](#data-privacy)
+   - [Untrusted environments or networks](#untrusted-environments-or-networks)
+   - [Multi-Tenant environments](#multi-tenant-environments)
+ - [**Reporting a Vulnerability**](#reporting-a-vulnerability)
+
+## Using AutoGPT Securely
+
+### Restrict Workspace
+
+Since agents can read and write files, it is important to keep them restricted to a specific workspace. This happens by default *unless* RESTRICT_TO_WORKSPACE is set to False.
+
+We do not recommend disabling RESTRICT_TO_WORKSPACE, unless AutoGPT is running in a sandbox environment where it couldn't do any damage (e.g. Docker or a VM).
```

Comment:
Can we safely say that AutoGPT "couldn't do any damage" in Docker alone?

Reply:
It can do damage in docker if it escapes and we don’t support preventing escapes

Reply:
That's right, if it escapes it could affect the host machine. And it seems there are [sandboxes safer than docker](https://news.ycombinator.com/item?id=10052163#:~:text=Docker%20Content%20Trust-,Unfortunately%2C%20no%2C%20Docker%20is%20not%20a%20good%20sandbox%2C%20because,API%20as%20its%20security%20boundary) so I've rewritten it and linked for a list of some sandboxes and its security aspects instead of explicitly mentioning docker.